pkg/apis/componentconfig/edgecore/v1alpha2/default.go

```diff
@@ -40,7 +40,6 @@ func NewDefaultEdgeCoreConfig() *EdgeCoreConfig {
 
 	in := kubeletconfigv1beta1.KubeletConfiguration{}
 	in.ContentType = "application/json"
-	in.NodeStatusUpdateFrequency = metav1.Duration{Duration: constants.DefaultNodeStatusUpdateFrequency}
```

Comment:
The const `NodeStatusUpdateFrequency` can also be removed？

Reply:
done